package.json

```diff
@@ -94,6 +94,7 @@
     "tslib": "^2.8.1",
     "typescript": "^5.8.2",
     "upath": "^2.0.1",
+    "vscode-jsonrpc": "^8.2.1",
```

Comment:
@neilcampbell @PatrickDinh i figured since it was already in transitive + we use pygls that is compliant with language server protocol on py side, would be easier to leverage that instead of dealing with manual handling of stdin, stdout buffers and assembling json rpc message.